bigbluebutton-html5/imports/ui/components/app/service.js

```diff
@@ -23,12 +23,12 @@ const clearModal = () => {
 };
 
 const getCaptionsStatus = () => {
-  const settings = SettingsService.getSettingsFor('cc');
-  return settings ? settings.closedCaptions : false;
+  const settings = Settings.cc;
```

Comment:
Not sure why, but it's always undefined here.
It works for the Settings.application below, but doesn't work here. Closed caption window never opens.